lib/metasploit/framework/login_scanner/ivanti_login.rb

```diff
@@ -0,0 +1,169 @@
+require 'metasploit/framework/login_scanner/http'
+
+module Metasploit
+  module Framework
+    module LoginScanner
+      # Ivanti Login Scanner supporting
+      # - User Login
+      # - Admin Login
+      class Ivanti < HTTP
```

Comment:
Looks like this is missing some of the boilerplate like `check_setup` for verifying the target is indeed ivanti

https://github.com/rapid7/metasploit-framework/blob/90ad8b66d8ac470012525d6d25c06e520b434590/lib/metasploit/framework/login_scanner/teamcity.rb#L144-L159

And some of the login scanner metadata - https://github.com/rapid7/metasploit-framework/blob/90ad8b66d8ac470012525d6d25c06e520b434590/lib/metasploit/framework/login_scanner/teamcity.rb#L123-L131

https://github.com/rapid7/metasploit-framework/blob/90ad8b66d8ac470012525d6d25c06e520b434590/lib/metasploit/framework/login_scanner/jenkins.rb#L8-L12